client/src/elm/Backend/ResilienceSurvey/Model.elm

```diff
@@ -37,6 +44,11 @@ type ResilienceSurveyQuestionOption
     | ResilienceSurveyQuestionOption2
     | ResilienceSurveyQuestionOption3
     | ResilienceSurveyQuestionOption4
+    | ResilienceSurveyQuestionOption5
```

Comment:
Currently, we support up to 4  answers for a question.
What's the maximal number of answers for question at adoption survey? 9?
If not, drop unneeded options.

Reply:
they are 5 and they are different from the 4 we had for the previous survey

Reply:
How are they different? If question has up to 5 answers, we should have 5 options.

Reply:
I get it